### DISCUSSION TITLE
有人能提供一下群晖的操作吗？

### DISCUSSION THREAD

**Discussion by mjyhj** [NewFuture/DDNS#246] [Q&A] [answered]
找了很多脚本都不支持dnspod的ipv6
终于找到了这个，但是不知道该怎么操作？

有人能提供一下详细的小白教程吗？

**Comment by juzhijian** [comment, marked as answer]
安装https://registry.hub.docker.com/r/newfuture/ddns/
找个目录创建一个config.json
然后把创建的config.json 连接到/config.json
网络使用 与docker相同网络。

**Comment by sagezhj** [comment]
如何查看docker内运行日志？容器运行正常，但是地址没有更新

**Comment by juzhijian** [comment]
> 如何查看docker内部运行日志？容器运行正常，但是地址没有更新

在详情里面有个终端机。放一会就有了。每次更新有时间的。
